command/command.go

```diff
@@ -210,3 +211,33 @@ func setEnvVar(c *cli.Command) {
 		setEnvVar(&c.Subcommands[i])
 	}
 }
+
+// FingerprintFormatFlag returns a flag for configuring the fingerprint format.
+func FingerprintFormatFlag() cli.StringFlag {
+	return cli.StringFlag{
+		Name:  "format",
+		Usage: `The <format> of the fingerprint, it must be "hex", "base64", "base64-url", "base64-raw", or "emoji".`,
+	}
```

Comment:
You can also add a `Value: "hex"` here to use it as default, so you can get rid of the defaultFmt in the next method.

But, you will need a second flag with `base64-raw` for `step ssh fingerprint` as the default one is base64.RawStdEncoding. See next comment.

Reply:
Done.